docs/Programming Framework.md

```diff
@@ -80,6 +80,8 @@ IPF can be edited using INAV Configurator user interface, of via CLI
 | 44            | MAX                           | Finds the highest value of `Operand A` and `Operand B` |
 | 45			| FLIGHT_AXIS_ANGLE_OVERRIDE	| Sets the target attitude angle for axis. In other words, when active, it enforces Angle mode (Heading Hold for Yaw) on this axis (Angle mode does not have to be active). `Operand A` defines the axis: `0` - Roll, `1` - Pitch, `2` - Yaw. `Operand B` defines the angle in degrees |
 | 46			| FLIGHT_AXIS_RATE_OVERRIDE	    | Sets the target rate (rotation speed) for axis. `Operand A` defines the axis: `0` - Roll, `1` - Pitch, `2` - Yaw. `Operand B` defines the rate in degrees per second |
+| 47            | EDGE                          | `Operand A` is activation operator [`boolean`], `Operand B` is the time for the edge to stay active [ms]. After activation, operator will return `true` until the time in Operand B is reached |
```

Comment:
@MrD-RC all good, but technically this is not `EGDE` but `LATCH` or `TIMED_LATCH` or `TEMPORARY_LATCH`. 
`EDGE` would be a detection if `OperandA` value changed more than `OperandB`. 

So maybe let's rename it to a more proper name. And BTW, `EDGE` might be useful too

Reply:
Wouldn't that be a delta **|a|>x** switch, if **A** changes more than **B**?

I always took edge to be a momentary switch, activated by operand **A**. I just added an optional delay to it

Either way, I can add the **A** changes more than **B** comparator too. Happy to name them whatever. 

I actually requested the review to make sure you were happy with how the switch is working, as I would like to add more 😁. x~=a and a simple on/off timer are next. I could even add a delay, similar to what is currently named edge, but the time is how long to wait before the switch activates. So long as **A** is true.